src/library/clementine/clementineplaylistmodel.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <QHash>
+#include <QtSql>
+
+#include "library/basesqltablemodel.h"
+#include "library/clementine/clementinedbconnection.h"
+#include "library/trackcollection.h"
```

Comment:
Use forward declarations where appropriate

Reply:
tried to add this in https://github.com/mixxxdj/mixxx/pull/3173/commits/f2a59de119ecefa97cf8d9ae8821a79d37650a05